config/initializers/salesforce_cache.rb

```
@@ -1,5 +1,10 @@
 # We clear the cache at startup so restarting the server
 # is an easy way to force an immediate update.
-SalesforceCache.all.each do |item|
-  item.destroy!
+begin
+  SalesforceCache.all.each do |item|
+    item.destroy!
+  end
+rescue
+  # failure is acceptable, this is non-essential
```

> Is there a way to log? Catching with no indication is a recipe for hard to troubleshoot behavior.

[11:51 AM] Adam Ruppe: that reminds me the other one is the salesforce cache but logging that is useless because the scenario is: empty the cache. it complains it is already empty.... so yay, that means you don't have to actually do anything!
		this happened when i ran rake test, since test starts with a totally empty database
[11:53 AM] Brian Sadler: we should have this conversation in github so that people can see it in the future, but in that case, it should only catch and swallow that exact exception.  catching all exceptions and suppressing them is dangerous